app/bundles/CoreBundle/Controller/FormController.php

```diff
@@ -116,7 +116,7 @@ protected function getRouteBase()
      */
     protected function getSessionBase($objectId = null)
     {
-        return $this->deprecatedSessionBase;
+        return null !== $this->deprecatedSessionBase ? $this->deprecatedSessionBase : parent::getSessionBase($objectId);
```

Comment:
Because some controllers extend FormController and some of them AbstractStandardFormController
We need unify return getSessionBase method if was not defined  by deprecatedSessionBase.
It was empty, even If we expect something.
Your suggestion is cycled

Reply:
Ok, thanks for the details